### ISSUE TITLE
No blue focus ring on images in array in document

### ISSUE THREAD

**Issue by thisisui** [sanity-io/sanity#2722]
When I am tabbing thought document I do not have visually indication of selected image in arrays. I can see blue outline on drag handle and remove icon but when I am tabbing through image I do not see it.

**To Reproduce**

Steps to reproduce the behaviour:

1. Create sanity project with E-commerce template
2. Click on first Product 
3. Tab to the Images section to the product when array is present
4. Tab through image, where no visual indication is on the small thumbnail

**Expected behaviour**

Default browser focus ring should be present

**Which versions of Sanity are you using?**

2.17.0

**What operating system are you using?**

OSX

**Which versions of Node.js / npm are you running?**

v14.16.0

**Comment by kmelve** [comment]
Hi! 

We are currently working on improving our workflows and follow-up on our open GitHub repository. In that work, we have decided to close most issues older than the release of Sanity Studio v3.

We value your feedback, so if this issue is still important to you and relevant for Sanity Studio v3, please search for relevant open issues. If you can’t find any, open a new one and link to relevant comments in this thread. For questions about how to do something, please post them in the [slack community](https://slack.sanity.io).
